joomla/joomla-cms
# [4.0] Namespace Editors - CodeMirror plugin

### Summary of Changes

Namespaces Editors - CodeMirror plugin.

### Testing Instructions

Delete `libraries/autoload_psr4.php` file.
Edit Editors - CodeMirror plugin.
In Appearance tab check that Font selector still appears.

### Documentation Changes Required

IDK.
